content/news/playtest-20160424.md

```diff
@@ -0,0 +1,80 @@
+We are happy to present to you the second playtest in this release cycle!
+
+Many users &ndash; indeed more than we anticipated (over 2000!) &ndash; have tried out the first playtest and gave us a lot of feedback. Almost inevitably, a number of bugs have also been reported (thanks to all the testers who took their time to do so!), many of them quite hard to track down.
```

Comment:
Could streamline this a bit with "a number of bugs have also been reported" &rarr; "a number of bugs were reported" and dropping the "many of them quite hard to track down".